opts/envfile_test.go

```diff
@@ -29,7 +29,12 @@ func TestParseEnvFileGoodFile(t *testing.T) {
 
 _foobar=foobaz
 `
-
+	// Adding a newline + a line with pure whitespace.
+	// This is being done like this instead of the block above
+	// because it's common for editors to trim trailing whitespace
+	// from lines, which becomes annoying since that's the
+	// exact thing we need to test.
+	content += "\n    "
```

Comment:
Doh! You're right, probably combined with the "trim leading" title of the PR, I got completely lost LOL

> I could just add a \t into this line as well to cover both

Saw you added it, thanks!